src/Pages/Login/Login.jsx

```diff
@@ -0,0 +1,268 @@
+import React from 'react';
+import { LOGINIMAGES } from './data/data';
+import { KAKAO_API } from '../../config';
+import { Link, useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { flexCenter } from '../../Styles/CommonStyle';
+
+const { Kakao } = window;
+
+const Login = () => {
+  const history = useHistory();
+  const goToKakao = () => {
+    Kakao.Auth.login({
+      success: function (authObj) {
+        console.log('정상 로그인 토큰 :', authObj.access_token);
+        fetch(`${KAKAO_API}`, {
+          method: 'POST',
+          body: JSON.stringify({
+            access_token: authObj.access_token,
+          }),
+        })
+          .then((res) => res.json())
+          .then((res) => {
+            console.log('res : ', res);
+            console.log('res.access_token : ', res.access_token);
+            localStorage.setItem('Kakao_token', res.access_token);
+            if (res.access_token) {
+              alert('SUWEE의 서재에 오신걸 환영합니다!');
+              history.push('/main');
+            }
+          });
+      },
+      fail: function (err) {
+        alert(JSON.stringify(err));
+      },
+    });
+  };
+
+  const goToLogOut = () => {
+    if (Kakao.Auth.getAccessToken()) {
+      console.log('카카오 인증 액세스 토큰이 존재함', Kakao.Auth.getAccessToken());
+      Kakao.Auth.goToLogOut(() => {
+        console.log('로그아웃됨', Kakao.Auth.getAccessToken());
+      });
+    }
+  };
+  return (
+    <StyleLogin>
+      <BackgroundImgContainer>
+        <LoginBoxContainer>
+          <img className="backgroundImg" src="/images/library.jpg" alt="배경화면" />
+        </LoginBoxContainer>
+      </BackgroundImgContainer>
+      <LoginContainer>
+        <LogoWrap>
+          <Wrap>
+            <LogoBox>
+              <div className="LogoImgBox">
+                <img src="/images/suweelogo.png" alt="로고 사진" />
+              </div>
+            </LogoBox>
+          </Wrap>
+          <InputPhoneId type="number" placeholder="휴대폰 번호"></InputPhoneId>
+          <InputPassword type="password" placeholder="비밀번호"></InputPassword>
+          <Button>휴대폰 번호로 로그인</Button>
+          <OrText>⎼⎼⎼⎼⎼⎼⎼⎼⎼⎼⎼⎼⎼⎼ 또는 ⎼⎼⎼⎼⎼⎼⎼⎼⎼⎼⎼⎼⎼⎼</OrText>
+          <SnsContainer>
+            {LOGINIMAGES.map((img) => {
+              if (img.alt === '카카오') {
+                return (
+                  <SnsBox onClick={goToKakao}>
+                    <img key={img.id} src={img.src} alt={img.alt} />
+                  </SnsBox>
+                );
+              } else
+                return (
+                  <SnsBox>
+                    <img key={img.id} src={img.src} alt={img.alt} />
+                  </SnsBox>
+                );
+            })}
```

Comment:
반복되는 코드가 보이네요! 
아래와 같이 작성해주시면 좀 더 간결해 질 수 있을 것 같습니다. 
```suggestion
            {LOGINIMAGES.map((img) => {
              return (
                <SnsBox onClick={img.alt === "카카오" ? goToKakao : ""}>
                  <img key={img.id} src={img.src} alt={img.alt} />
                </SnsBox>
              );
            })}
```

Reply:
헐 이렇게 줄일수 있다니!!! 너무 감사합니다 ㅜㅜㅜ